owa/app/js/components/manage_patient_identifier_sources/new_patient_identifier_sources/localIdentifierGenerator.jsx

```diff
@@ -0,0 +1,85 @@
+import React, {Component} from 'react';
+import {Table, Button} from 'reactstrap';
+import './newPatientIdentifierSources.css'
+
+export default class LocalIdentifierGenerator extends Component {
+
+  constructor(props) {
```

Comment:
You do not need this constructor, it's useless here

Reply:
fixed